tests/integration_tests/functional/test_signals.py

```diff
@@ -58,6 +66,50 @@ def test_sigbus_sigsegv(test_microvm_with_api, signum):
     assert metric_line["signals"][signum_str[signum]] == 1
 
 
+def test_sigxfsz_handler(test_microvm_with_api):
+    """Test intercepting and handling SIGXFSZ."""
+    microvm = test_microvm_with_api
+    microvm.spawn()
+
+    # We don't need to monitor the memory for this test.
+    microvm.memory_monitor = None
```

Comment:
This might not be needed.

Reply:
It seems to be needed, the `monitor_memory` parameter defaults to `True` when creating a new microvm: https://github.com/firecracker-microvm/firecracker/blob/master/tests/framework/microvm.py#L57.

Reply:
Ok. But what happens if the memory monitor is running ? We shouldn't consume too much memory for this test.

Reply:
A new thread is created for monitoring the RSS memory usage while Firecracker runs. The reason for disabling it is that it seemed irrelevant for what we are testing. Do you think it is worth enabling it for this test?

Reply:
Ok, I see. Then I don't have preferences. I just didn't know why the `memory_monitor` was disabled. I thought it was an assumption that it would impact the test.